core/src/main/java/com/aem/demo/core/models/MainPageModel.java

```diff
@@ -56,7 +55,7 @@ private void init() {
             return;
         }
         models = new ArrayList<>();
```

Comment:
Please remember the comment from the previous PR that assigning a new ArrayList to `models` here is senseless. The value will in every case be reassigned in either line 85, 89, or 92. Please do not ignore the PR comments